README.md

```diff
@@ -77,12 +123,6 @@ The following environment variables are used to configure the Hydrolix connectio
 * `HYDROLIX_VERIFY`: Enable/disable SSL certificate verification
   - Default: `"true"`
   - Set to `"false"` to disable certificate verification (not recommended for production)
-* `HYDROLIX_CONNECT_TIMEOUT`: Connection timeout in seconds
-  - Default: `"30"`
-  - Increase this value if you experience connection timeouts
-* `HYDROLIX_SEND_RECEIVE_TIMEOUT`: Send/receive timeout in seconds
-  - Default: `"300"`
-  - Increase this value for long-running queries
```

Comment:
If these are being ignored, should they be removed from the code as well?

Reply:
I'd like to leave them in the code for now to see if mcp-clickhouse chooses to resolve the issue. They just get piped to some clickhouse-client settings, but as long as the values are >=30s, the MCP tool execution will kill itself before these timeouts take effect anyways.

I don't have a strong enough model of how clickhouse-client timeout works to know whether the issue is fixable, but if it is, leaving the backing code in will make it easier to port the fix from upstream.